src/main.rs

```diff
@@ -34,17 +32,23 @@ fn main() {
     let mut line_file = Vec::<u8>::new();
     f.read_to_end(&mut line_file).unwrap();
 
-    let max_size_file = 1024 * 1024; // 1mb, or 1024 kilobytes
-    assert!(max_size_file >= line_file.len());
+    // let max_size_file = 1024 * 1024; // 1mb, or 1024 kilobytes
+    // assert!(max_size_file >= line_file.len());
```

Comment:
I think we tried to build some minimal check to avoid zip-bomb like file parsing here.